XiEditor/EditView.swift

```diff
@@ -166,9 +114,15 @@ class EditView: NSView, NSTextInputClient {
         path2.fill()
         */
 
+        let defaultStyle = dataSource.styleMap.defaultStyle
+
+        // set view background
+        self.wantsLayer = defaultStyle.bgColor.alphaComponent != 0.0
+        self.layer?.backgroundColor = defaultStyle.bgColor.cgColor
```

Comment:
Also see my comment about `setTextBgColor`: the `bgColor` field of a style should not be interpreted as the view's background color, as fun as having that working is. 😢